src/components/MultiEmailInput/constants.jsx

```diff
@@ -42,6 +46,36 @@ const MultiValueRemove = props => (
   </components.MultiValueRemove>
 );
 
+const CustomValueContainer = ({ children, ...props }) => {
+  const {
+    getValue,
+    selectProps: { isFocused },
+  } = props;
+  const value = getValue();
+  const newChildren = [...children];
```

Comment:
Since its a shallow copy

```suggestion
  const [firstChild, ...rest] = children;
```